---
bool success
string message
dcsc_setups/PendulumSensors sensors